src/main/java/thePackmaster/SpireAnniversary5Mod.java

```diff
@@ -324,6 +326,26 @@ public String glowID() {
         addPotions();
     }
 
+
+    public static void addPotions() {
+
+        BaseMod.addPotion(AttackPotionButClaw.class, Color.RED, Color.WHITE, Color.FIREBRICK, AttackPotionButClaw.POTION_ID);
+        BaseMod.addPotion(ClawPowerPotion.class, Color.RED, Color.WHITE, Color.FIREBRICK, ClawPowerPotion.POTION_ID);
+        BaseMod.addPotion(DrawClawsPotion.class, Color.RED, Color.WHITE, Color.FIREBRICK, DrawClawsPotion.POTION_ID);
+        BaseMod.addPotion(GenerateClawsPotion.class, Color.RED, Color.WHITE, Color.FIREBRICK, GenerateClawsPotion.POTION_ID);
+
+        /*
+        //TODO - Just before ship, add this cause otherwise we'll trigger a dependency for everyone.
```

Comment:
As a separate follow up, I have a way to do this wide potions stuff that does not require any build dependency at all (and I think we should avoid build dependencies to make it as easy as possible for contributors to build this). After all, it sounds like there are people who want to contribute packs after release, and I'm sure even people who have finished their packs will want to tweak things.

So no need to worry about it for now, I'll make a future PR that changes this.